processor/cumulativetodeltaprocessor/README.md

```diff
@@ -1,33 +1,72 @@
 # Cumulative to Delta Processor
-**Status: under development; Not recommended for production usage.**
```

Comment:
Not sure that this change is appropriate for this PR.  Even if we think that this makes the component ready for production use we should handle that separately from this functional change.

Reply:
Thats fair,  I will add it back.